openapi/p4pa-pu-bff.openapi.yaml

```
@@ -0,0 +1,106 @@
+openapi: 3.0.1
+info:
+  title: API Documentation
+  description: OpenAPI documentation
+  version: 1.0.0
+servers:
+  - url: "http://localhost:8080"
+    description: Generated server url
+paths:
+  /organizations:
+    get:
+      tags:
+        - Organizations
+      summary: "Retrieve the list of organizations"
+      operationId: getOrganizations
+      security:
+        - bearerAuth: [ ]
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/OrganizationDTO'
+        '401':
+          description: Unauthorized - Missing or invalid token
+        '403':
+          description: Forbidden - Access denied
+        '500':
+          description: Internal Server Error
+  /info/brokerConfig:
+    get:
+      tags:
+        - ConfigFe
+      summary: "Retrieve application information for FE"
+      operationId: getBrokerConfig
+      security:
+        - bearerAuth: [ ]
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ConfigFE'
+        '401':
+          description: Unauthorized - Missing or invalid token
+        '403':
+          description: Forbidden - Access denied
+        '500':
+          description: Internal Server Error
+components:
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+  schemas:
+    OrganizationDTO:
+      type: object
+      properties:
+        orgId:
+          type: string
+          description: Organization ID
+        name:
```

let's align field names with the model:
https://github.com/pagopa/p4pa-payhub-activities/blob/main/src/main/java/it/gov/pagopa/payhub/activities/dto/OrganizationDTO.java
```suggestion
        orgName:
```